mockup/patterns/relateditems/pattern.js

```diff
@@ -99,27 +99,30 @@ define([
       homeText: _t('home'),
       folderTypes: ['Folder'],
       selectableTypes: null, // null means everything is selectable, otherwise a list of strings to match types that are selectable
-      attributes: ['UID', 'Title', 'portal_type', 'path', 'getIcon'],
+      attributes: ['UID', 'Title', 'portal_type', 'path','getURL', 'getIcon','contenttype_class','is_folderish','review_state'],
```

Comment:
attributes are meant to be metadata in the catalog. It seems like an abuse to do those custom non-metadata fields in python. It can just as easily be calculated in javascript. We can add a utility method to the utils.js file if we need it.